MaterialDesignThemes.Wpf/DrawerClosingEventHandler.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace MaterialDesignThemes.Wpf
+{
+    public delegate void DrawerClosingEventHandler(object sender, DrawerClosingEventArgs eventArgs);
```

Comment:
Rather than creating your own delegate we can simply use `EventHandler<DrawerClosingEventArgs>`